CanHazFunny/CanHazFunny.Tests/OutputJokeTests.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using Assert = NUnit.Framework.Assert;
+using System.IO;
+using NUnit.Framework;
+
+namespace CanHazFunny.Tests;
+public class OutputJokeTests
+{
+    [Test]
+    public void OutputJoke_toConsole()
+    {
+        var outputJoke = new OutputJoke();
+        var aJoke = "This is a joke";
```

Comment:
```suggestion
        OutputJoke outputJoke = new();
        string aJoke = "This is a joke";
```
Might be wrong about this but I think they wanted us to avoid using var when initializing variables and such, what you guys had originally still works though.

Reply:
I'll change it I honestly don't recall

Reply:
`var` should only be used when the type is obvious, such as the aJoke. Even then, it's usually recommended to specify the type over the use of `var`.